electrical/breakbeam.py

```diff
@@ -0,0 +1,47 @@
+import RPi.GPIO as GPIO
+import time
+
+class Breakbeam:
+    ACCEPTABLE_TIME = 50 # in seconds
+    BEAM_PINS = {17:'50%_1', 22:'50%_2', 23:'100%_1', 24:'100%_2'}
+
+    def __init__(self):
+        for BEAM_PIN in Breakbeam.BEAM_PINS.keys():
+            GPIO.setmode(GPIO.BCM)
+            GPIO.setup(BEAM_PIN, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+            GPIO.add_event_detect(BEAM_PIN, GPIO.BOTH, callback=self.break_beam_callback)
+            print(self.timer(BEAM_PIN))
+        self.beam_broken = False
+        self.blocked_sensors = []
```

Comment:
also do we care about the order theyre blocked in? if not a set would be more efficient here + removes possibilities of duplicates being added

Reply:
is there a minimum number of sensors that need to be blocked in order for us to deduce that we're 50% filled / 100% filled?